src/NuGet.Core/NuGet.LibraryModel/LibraryIncludeFlagUtils.cs

```diff
@@ -24,6 +24,14 @@ public static class LibraryIncludeFlagUtils
         /// Convert set of flag strings into a LibraryIncludeFlags.
```

Comment:
Team Review: Update this XML doc comment to say it treats `BuildTranstive` as `BuildTranstive | Build`